samples/keyboard/keyboard.cpp

```diff
@@ -386,7 +386,9 @@ const char* GetVirtualKeyCodeName(int keycode)
         WXK_(RBUTTON)
         WXK_(CANCEL)
         WXK_(MBUTTON)
-        WXK_(CLEAR)
+#ifdef __WXMSW__
+        WXK_(NUMPAD_CENTER) // = WXK_CLEAR on MSW
+#endif
```

Comment:
Do we really need preprocessor checks here? I.e. why can't we just use `WXK_(NUMPAD_CENTER)` once for all ports?

Reply:
fixed, please review